src/lightly_train/_logging.py

```diff
@@ -157,7 +157,7 @@ def set_up_file_logging(log_file_path: Path) -> None:
     ]
 
     for file_logger in file_loggers:
-        _remove_handlers(file_logger, LightlyTrainRotatingFileHandler)
+        _remove_handlers(file_logger, logging.FileHandler)
```

Comment:
Why is this change needed? We added `LightlyTrainRotatingFileHandler` to have a special class just for Lightly and not remove any other handlers that users might have attached.